Лабораторная работа 3

zotovaad opened the pull request.

zotovaad committed:
Лабораторная работа 3

vladuxa23 reviewed (approved):
Task1:

- Всё верно.

Task 6:

- Решено верно. В следующий раз в подобной задаче надо использовать кортежное присваивание (a, b = b, a), таким образом сможем избавиться от переменной temp

Task 8:

- Не представлено
 
Task 9:

- Не представлено

zotovaad committed:
Лабораторная работа 3

Добрый вечер! 
Сегодня я добавила к лабораторной 3 файлы с task_8 и task_9
Ссылку на этот Pull request повторно пришлю в СДО

zotovaad commented:
Task 6: Спасибо за комментарий

Добавила Task 8 и Task 9 в файлы

vladuxa23 reviewed (approved):
Посмотрел, всё корректно, замечаний нет

zotovaad commented:
> Посмотрел, всё корректно, замечаний нет

Спасибо!
